### ISSUE TITLE
Introduce Configurations and Metacello in Chapter 8 or elsewhere

### ISSUE THREAD

**Issue by laheadle** [SquareBracketAssociates/UpdatedPharoByExample#123]
Pharo is used by software developers, and software developers depend on packages. Therefore I believe that one of the first things a new Pharo user will want to do is install packages written by others. But Pharo By Example contains no mention of the ConfigurationOfXYZ method of installing packages (and does not mention the configuration browser -- an awesome tool for new users to get excited about what is out there). You could introduce these topics without getting too deep into Metacello, but...

I also think Metacello should be introduced fairly early. Any application will almost certainly have dependencies and it is important to be able to load your application into a fresh image. Metacello is essential for this, so it is essential for Pharo.

Chapter 8 could probably be expanded to include these topics, which are more fundamental than Morphic or Seaside. Note that the Seaside chapter recommends installing an old one-click image! This is a great chance to use Configurations.

What do you think of this idea? And yes, I realize I can contribute this myself, but I currently decline the commitment. Apologies...

**Comment by DamienCassou** [comment]
I agree we should have some text about the configuration browser. Metacello is covered in DeepIntoPharo. We could only mention it in UpdatedPharoByExample and send the interested reader to DeepIntoPharo.

**Comment by dmitrizagidulin** [comment]
Agreed that we should introduce the Configuration browser early on. (And, yeah, Metacello can be left for Deep Into Pharo. Using packages is pretty fundamental, writing/editing packages is definitely an advanced topic).

**Comment by kilon** [comment]
Metacello should not live in a separate book since its fundamental feature that begineer pharo coders will depend on to create new projects and to load other projects. Deep Into Pharo can still be referenced as in detail doc for Metacello. I will create a seperate chapter for handling of project documenting both the Smalltalkhub and Github workflow with mentions of Versioners, Configurations, filetree, gitfiletree and of course Baselines. A brief description about the loading of configurations will be added to Pharo Tour chapter with a link to the chapter that will go deeper on this subject.

**Comment by dmitrizagidulin** [comment]
Sure, agreed. Although, why create a separate chapter? All of the things you mention (Versioner, SH and GH, etc) all belong in the existing Source Control and Sharing Code chapter.

**Comment by kilon** [comment]
ah my bad I forgot about that chapter :dancer: 
Yeah of course it should go in that chapter to connect with the rest of documentation and I see that chapter need an update anyway because the images look very old.

**Comment by dmitrizagidulin** [comment]
Perfect, thanks! :)
